README.md

```diff
@@ -42,11 +99,27 @@ Our Recommended template for React Native projects.
 
    Close your terminal and open a new one.
 
-4. Install GraphicsMagick `brew install graphicsmagick`.
-5. Run `bundle install`.
-6. Run the setup script: `node ./setup`. A successful setup will give you the green light: `✅ Setup completed!`.
+3. Install GraphicsMagick: `brew install graphicsmagick`. (used to generate splash screens)
+4. Install [Cocoapods](https://guides.cocoapods.org/using/getting-started.html#installation): `sudo gem install cocoapods`
+
+### Usage
 
----
+In order to use this template, follow these steps:
+
+1. Initialize a new react-native app: `react-native init MyApp --template=https://github.com/echobind/react-native-template-echobind`
+2. Change directories into your new app: `cd MyApp`
+3. Run `bundle install`
+4. Run the setup script: `node ./setup`. A successful setup will give you the green light: `✅ Setup completed!`
+
+## CI / Deployment:
+
+In order to set up CI/Deployment, we've created custom hygen templates to save you time. These live in our [`hygen-templates-echobind` repository](https://github.com/echobind/hygen-echobind-templates). You'll find installation instructions, which will allow you to set up the following:
+
+- CircleCI preconfigured for React Native. Fast build times!
+- Fastlane lanes for alpha, beta, and production releases. These should be run on CI only!
+- AppCenter integration for easy alpha distribution
+
+For more info, please follow the [instructions](https://github.com/echobind/hygen-echobind-templates) in that repo.
```

Comment:
looks like we also have some CI docs in `docs/circle-ci`. Should we move some of this there? What's the relation to the local hygen templates (in `_templates`) vs the `hygen-echobind-templates` repo?

Reply:
Good points! I address some of this in PR #56 - I'll go there and make an update related to `docs/circle-ci` as well.